src/Appwrite/Auth/Auth.php

```diff
@@ -43,6 +43,12 @@ class Auth
     public const USER_ROLE_APPS = 'apps';
     public const USER_ROLE_SYSTEM = 'system';
 
+    /**
+     * Audit User Types.
+     */
+    public const AUDIT_TYPE_APP = 'app';
```

Comment:
Not audit type, but user type, its always best to try and decouple libraries from knowing too much about each other and let the controller/action layer connect them. This help with better maintainability long terms and helps us keep better domain driven design.

Reply:
Also, with new DB we should lean to start stop using the term audit, and use activity. Eventually, we will also deprecate the audit library.

Reply:
Renaming to `ACTIVITY_TYPE_*` makes sense. Using `USER_TYPE_*` wouldn’t align with cases like `USER_TYPE_USER`.